crates/luminol-ui/src/tabs/started.rs

```diff
@@ -0,0 +1,199 @@
+
+        if let Some(p) = self.load_filesystem_promise.take() {
+            match p.try_take() {
+                Ok(Ok(host)) => {
+                    #[cfg(target_arch = "wasm32")]
+                    {
+                        idb_key = host.idb_key().map(str::to_string);
+                    }
+
+                    filesystem_open_result = Some(update_state.filesystem.load_project(
+                        host,
+                        update_state.project_config,
+                        update_state.global_config,
+                    ));
+                }
+                Ok(Err(error)) => update_state.toasts.error(error.to_string()),
+                Err(p) => self.load_filesystem_promise = Some(p),
+            }
+
+            ui.spinner();
+        }
+
+        ui.add_enabled_ui(self.load_filesystem_promise.is_none(), |ui| {
+            if ui
+                .button(egui::RichText::new("New Project").size(20.))
+                .clicked()
+            {
+                update_state
+                    .edit_windows
+                    .add_window(crate::windows::new_project::Window::default());
+            }
+            if ui
+                .button(egui::RichText::new("Open Project").size(20.))
+                .clicked()
+            {
+                // maybe worthwhile to make an extension trait to select spawn_async or spawn_local based on the target?
+                #[cfg(not(target_arch = "wasm32"))]
+                {
+                    self.load_filesystem_promise = Some(poll_promise::Promise::spawn_async(
+                        luminol_filesystem::host::FileSystem::from_file_picker(),
+                    ));
+                }
+                #[cfg(target_arch = "wasm32")]
+                {
+                    self.load_filesystem_promise = Some(poll_promise::Promise::spawn_local(
+                        luminol_filesystem::host::FileSystem::from_folder_picker(),
+                    ));
+                }
+            }
+        });
+
+        ui.add_space(100.);
+
+        ui.heading("Recent");
+
+        // FIXME
+        for path in update_state.global_config.recent_projects.clone() {
+            #[cfg(target_arch = "wasm32")]
+            let (path, idb_key) = path;
+
+            if ui.button(&path).clicked() {
+                let path = path.clone();
+
+                #[cfg(not(target_arch = "wasm32"))]
+                {
+                    filesystem_open_result = Some(update_state.filesystem.load_project_from_path(
+                        update_state.project_config,
+                        update_state.global_config,
+                        path,
+                    ));
+                }
+
+                #[cfg(target_arch = "wasm32")]
+                {
+                    self.load_filesystem_promise = Some(poll_promise::Promise::spawn_local(
+                        luminol_filesystem::host::FileSystem::from_idb_key(path),
+                    ));
+                }
```

Comment:
In order for loading from the recent projects menu to work in web, `from_idb_key` needs to be passed `idb_key`, not `path`.